konflux-ci/fbc/catalog.ocp5.Dockerfile

```
@@ -0,0 +1,25 @@
+ARG OCP_V=latest
+
+# The builder image is expected to contain
+# /bin/opm (with serve subcommand)
+FROM registry.redhat.io/openshift5/ose-operator-registry-rhel9:$OCP_V as builder
+
+# Copy FBC root into image at /configs and pre-populate serve cache
+ADD bundle.object/catalog /configs
```

Hadolint (2.15.1)

[error] Use COPY instead of ADD for files and folders

(DL3020)